lib/console_user_interface.rb

```diff
@@ -7,12 +7,128 @@ class ConsoleUserInterface
   def initialize(output_stream:, input_stream:)
     @output_stream = output_stream
     @input_stream = input_stream
-    @prior_guess_result = nil
+    @status = ""
   end
 
-  def show_potential_fleet_board(fleet_board)
+  def rows
+    ["a", "b", "c", "d", "e", "f", "g", "h", "i", "j"]
+  end
+  
+  def clear_view
     system("cls") || system("clear")
-    show_fleet_board(fleet_board)
+  end
+
+  def show_potential_fleet_board(fleet_board)
+    clear_view
+    str = line
+    str += "|                          FLEET BOARD                            |\n" 
```

Comment:
If these white-spaces are constant, it might make sense to extract them to constants and share them.

```
TEN_BLANKS = "         "
...
str += "|#{TEN_BLANKS}FLEET BOARD#{TEN_BLANKS}|\n"